subprojects/testfx-core/testfx-core.gradle

```diff
@@ -21,12 +21,14 @@ ext.pomName = 'TestFX Core'
 ext.pomDescription = 'TestFX core library'
 ext.moduleName = 'org.testfx'
 ext.openjfxVersion = '23.0.1'
+ext.headlessSDKPath= "${System.properties['user.home']}${File.separator}Downloads${File.separator}javafx-sdk-24"
```

Comment:
this is a rather strong requirement. Can't we just take an environment variable here? (e.g. System.getEnv("JAVAFX_HOME"))

Reply:
makes sense, done